<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtnomecliente">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtcpf">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtendereco">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lblnomecliente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblcpf">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPF:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblendereco">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Endereço:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblnumero">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>550</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nº:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtnumero">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrarCliente">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>680</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(130, 162, 37);</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Inserir_um_título-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblFundo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>781</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 214, 214);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtidanimal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>620</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="lblidanimal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID Animal:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txttelefone">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbltelefone">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Telefone:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblemail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtemail">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>480</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAtualizarCliente">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>680</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Atualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnvoltarmenucliente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(165, 165, 165);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>botao-voltar.png</normaloff>botao-voltar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnlimpar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>680</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <zorder>lblFundo</zorder>
   <zorder>txtnomecliente</zorder>
   <zorder>txtcpf</zorder>
   <zorder>txtendereco</zorder>
   <zorder>lblnomecliente</zorder>
   <zorder>lblcpf</zorder>
   <zorder>lblendereco</zorder>
   <zorder>lblnumero</zorder>
   <zorder>txtnumero</zorder>
   <zorder>btnCadastrarCliente</zorder>
   <zorder>lblLogo</zorder>
   <zorder>txtidanimal</zorder>
   <zorder>lblidanimal</zorder>
   <zorder>txttelefone</zorder>
   <zorder>lbltelefone</zorder>
   <zorder>lblemail</zorder>
   <zorder>txtemail</zorder>
   <zorder>btnAtualizarCliente</zorder>
   <zorder>btnvoltarmenucliente</zorder>
   <zorder>btnlimpar</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
